phones/management/commands/delete_phone_data.py

```diff
@@ -0,0 +1,118 @@
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from typing import Any
+
+from django.core.management.base import BaseCommand, CommandError
+
+from allauth.socialaccount.models import SocialAccount
+
+from phones.models import InboundContact, RealPhone, RelayNumber
+
+
+@dataclass
+class PhoneData:
+    """Holds phone data for a user."""
+
+    fxa: SocialAccount
+    real_phone: RealPhone | None = None
+    relay_phone: RelayNumber | None = None
+    inbound_contact_count: int = 0
+
+    @classmethod
+    def from_fxa(cls, fxa_id: str) -> "PhoneData":
+        """Initialize from an FxA ID."""
+        fxa = SocialAccount.objects.get(provider="fxa", uid=fxa_id)
+
+        try:
+            real_phone = RealPhone.objects.get(user=fxa.user)
+        except RealPhone.DoesNotExist:
+            return cls(fxa=fxa)
+
+        try:
+            relay_phone = RelayNumber.objects.get(user=fxa.user)
+            inbound_contact_count = InboundContact.objects.filter(
+                relay_number=relay_phone
+            ).count()
+        except RelayNumber.DoesNotExist:
+            return cls(fxa=fxa, real_phone=real_phone)
+
+        return cls(
+            fxa=fxa,
+            real_phone=real_phone,
+            relay_phone=relay_phone,
+            inbound_contact_count=inbound_contact_count,
+        )
+
+    @property
+    def has_phone_data(self) -> bool:
```

Comment:
suggestion (non-blocking): `has_phone_data` seems a bit ambiguous on a `PhoneData` class. Maybe this should be `has_real_phone`?

Reply:
Or if it's meant to return `True` so long as the user has *ANY* phone data, maybe `has_data`?